wpeverest/user-registration
# Enhance - Review notice content and design

### All Submissions:






### Changes proposed in this Pull Request:
Previously the content and the design is not appropriate, so the content and the design have been changed in this PR. 

### How to test the changes in this Pull Request:

1. Verify if the contents and design of the review notice have been changed.
![download](https://user-images.githubusercontent.com/24367265/113555839-415f9a80-961b-11eb-98fb-0031a8641011.png)

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Review notice content and design.
